components/admin_console/feature_discovery/feature_discovery.scss

```diff
@@ -0,0 +1,46 @@
+@import 'utils/variables';
+
+.FeatureDiscovery {
+    background: var(--center-channel-bg);
+    box-shadow: 0px 2px 3px rgba(0, 0, 0, 0.08);
+    border-radius: 8px;
+    padding: 20px;
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: normal;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(440px, 1fr));
```

Comment:
Thanks @michaelgamble for tipping me off to the `grid-template-columns` property.